components/SkillList.tsx

```diff
@@ -496,7 +496,7 @@ export function SkillList(props) {
 	const [searchText, setSearchText] = useState('');
 
 	useEffect(function () {
-		if (props.isOpen && searchInput.current) searchInput.current.focus();
+		if (props.isOpen && searchInput.current) searchInput.current.select();

```

Comment:
Per [MDN](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/select#notes) we probably have to use both?